externals/sha2/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(SHA2)
```

Comment:
Actually, I think we don't need that dependency at all anymore.  We should just get rid of it.

Reply:
I will try to just remove this then.